cmd/ebrelayer/contract/generated/bin/BridgeBank/CosmosBank.bin

```diff
@@ -0,0 +1 @@
+608060405234801561001057600080fd5b506000808190555060006001819055506102518061002f6000396000f3fe608060405234801561001057600080fd5b50600436106100415760003560e01c8063328470ab14610046578063416d322714610064578063ebb73ca914610082575b600080fd5b61004e61017d565b6040518082815260200191505060405180910390f35b61006c610183565b6040518082815260200191505060405180910390f35b61013b6004803603602081101561009857600080fd5b81019080803590602001906401000000008111156100b557600080fd5b8201836020820111156100c757600080fd5b803590602001918460018302840111640100000000831117156100e957600080fd5b91908080601f016020809104026020016040519081016040528093929190818152602001838380828437600081840152601f19601f820116905080830192505050505050509192919290505050610189565b604051808273ffffffffffffffffffffffffffffffffffffffff1673ffffffffffffffffffffffffffffffffffffffff16815260200191505060405180910390f35b60005481565b60015481565b60006002826040518082805190602001908083835b602083106101c1578051825260208201915060208101905060208303925061019e565b6001836020036101000a038019825116818451168082178552505050505050905001915050908152602001604051809103902060009054906101000a900473ffffffffffffffffffffffffffffffffffffffff16905091905056fea265627a7a723158202ecb1c33f695b40bb02b295cdc00c550cc699b520554e2526ea8caa0dab5b80664736f6c63430005110032
```

Comment:
If the same version solc, the EVM bytecode should be same, it not depend on the host. As Luke explained, we check the bin in the repo because some people have problem during install the geth, abigen and solc. It will accelerate the manual test for now. We can remove them later.

Reply:
@juniuszhou please add an issue to remove later

Reply:
https://github.com/Sifchain/sifnode/issues/143 was created.